src/main/java/tasks/Task.java

```diff
@@ -0,0 +1,44 @@
+package tasks;
+
+public abstract class Task {
+    private String description;
+    private boolean isDone;
+
+    public Task(String description) {
+        setDescription(description);
+        setDone(false);
+    }
+
+    public Task(boolean done, String description) {
+        setDescription(description);
+        setDone(done);
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    public boolean isDone() {
+        return isDone;
+    }
+
+    public void setDone(boolean done) {
+        isDone = done;
+    }
+
+    public String getStatusIcon() {
+        return isDone() ? "[\u2714]" : "[\u2716]";
```

Comment:
Perhaps use a constant/enum for these?
Most (if not all) people will not know the identity of these unicodes off the top of their heads.

I know that that the IDE probably render these, but for the benefit of the reviewer and if you'd use these symbols in more places later in the future.

Reply:
Noted, i will add a constant.